### ISSUE TITLE
Plase add package.lock.json or yarn.lock

### ISSUE THREAD

**Issue by zerdos** [jamstack-cms/jamstack-cms#10]
Without it npm installs different versions, and the project might be broken with random error messages.

For example:
```
not finished load plugins - 0.671s
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gatsby-starter-blog@0.1.0 jamstack-develop: `node createConfig.js && gatsby develop`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the gatsby-starter-blog@0.1.0 jamstack-develop script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-11-05T12_00_51_245Z-debug.log
```

Caused by:
https://github.com/aws-amplify/amplify-js/issues/4268

**Comment by Glazy** [comment]
The project already contains a `yarn.lock` file 👍 

https://github.com/jamstack-cms/jamstack-cms/blob/master/yarn.lock

**Comment by dabit3** [comment]
I think you may have gotten this issue if you did not either `npm install` or `yarn` before running one of those commands? Are you still having this issue @zerdos ?

**Comment by zerdos** [comment]
yes, maybe because of the README.md - in the steps - there is ```npm install``` there.
My earlier experience - with gatsby and amplify tools - its better to use npm - for whatever reason.

I still coulnd't make it work, I will open more PR-s with the other issues.

**Comment by reece2208** [comment]
Same issue here

**Comment by dabit3** [comment]
@reece2208 have you all tried yarn?

**Comment by tonmaz** [comment]
**Same issue here**
![error](https://user-images.githubusercontent.com/54241333/68289408-57fc6d80-0086-11ea-9cde-1bc8886cb254.JPG)

**Comment by dabit3** [comment]
Hey, so tested this out some, and I have figured out that there are some issues with some npm dependencies with Amplify at the time being. For now, just use yarn and follow the new instructions, you should be good to go. Once I get the issues figured out with npm and the new release of Amplify is out, I'll also add a `package-lock.json` for npm users.
